Intersect.Server/Database/GameData/Migrations/BonusEffectsToListMigration.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Intersect.Logging;
```

Comment:
This migration is not needed, we can copy the old values via SQL rather than code